tools/submission/submission-checker.py

```diff
@@ -739,8 +739,13 @@ def log_result(submitter, available, division, system_type, system_name, system_
         if system_json.get("sw_notes"):
             notes = notes + ". " + system_json.get("sw_notes")
 
+        if system_type == "datacenter,edge" or system_type == "edge,datacenter":
```

Comment:
There are some checks on system type later in the code, ie we fetch the required scenarios by system type. And that code is needs to see  a specific scenario.
I think the easiest fix would be to add a:
```
for system_type in system_json.get("system_type").split(","): ...
```
above
```
for model_name in list_dir(results_path, system_desc): ...
```
That would take of everything with a single line.